addons/microsoft_calendar/models/calendar.py

```diff
@@ -74,7 +74,7 @@ def _skip_send_mail_status_update(self):
         """If microsoft calendar is not syncing, don't send a mail."""
         user_id = self._get_event_user_m()
         if self.with_user(user_id)._check_microsoft_sync_status() and user_id._get_microsoft_sync_status() == "sync_active":
-            return True
+            return not (self.user_id and self.user_id.partner_id not in self.partner_ids)
```

Comment:
Would prefer avoiding double negations as it's a bit confusing imo, isn't this condition sufficient?
"Skip emails if the organizer is an attendee"
`self.user_id.partner_id in self.partner_ids`

Reply:
That would make sense! Thanks